### PR TITLE
Refs #12913 - Turning on duplication checking cops

### PR DESCRIPTION
**Pull request by daviddavis**
Turning on cops that check for code duplication.

### CODE REVIEW

**Comment by komidore64** [comment]
:fire: :fire: :fire: :fire: :fire: :fire: :fire: :fire:

**Comment by komidore64** [comment]
ACK

File: app/models/katello/content_view.rb [hunk 1]
```diff
@@ -197,11 +197,6 @@ def puppet_env(env)
       ContentViewPuppetEnvironment.where(:id => ids).first
     end
 
-    def  puppet_repos
```
Comment (komidore64) [inline app/models/katello/content_view.rb]: does this method exist somewhere else?

Comment (daviddavis) [inline app/models/katello/content_view.rb]: Yep, [line 177](https://github.com/daviddavis/katello/blob/temp/20151222135922/app/models/katello/content_view.rb#L177-L180).

Comment (komidore64) [inline app/models/katello/content_view.rb]: perfect!

**Comment by komidore64** [comment]
ACK

**Pull request merged**